src/index.js

```diff
@@ -66,20 +56,24 @@ const ALL_COMMANDS = new Set([
 // Interprets the user's commands on Discord
 let messageAt = 0;
 bot.on('message', (msg) => {
+  // Bot sends nothing if user sends a non-command
   if (!msg.content.startsWith(COMMAND_PREFIX)) { return; }
```

Comment:
`startsWith()` is pretty neat! I wish I had taken advantage of this earlier haha.

Reply:
Yeah there's a lot of functions built in, but until you see them it's hard to know of them.